assets/js/modules/analytics-4/components/audience-segmentation/dashboard/AudienceTilesWidget/AudienceTiles.js

```diff
@@ -418,7 +362,40 @@ export default function AudienceTiles( { Widget, widgetLoading } ) {
 		}
 
 		return [ toClear, visible ];
-	}, [ audiences, configuredAudiences, dismissedItems, report ] );
+	}, [
+		audiences,
+		configuredAudiences,
+		dismissedItems,
+		isSiteKitAudiencePartialData,
+		report,
+		siteKitAudiences,
+		siteKitAudiencesReport,
+	] );
+
+	function checkForAllTilesError() {
+		const isSiteKitAudience = visibleAudiences.some(
+			( audienceResourceName ) =>
+				siteKitAudiences.some(
+					( audience ) => audience.name === audienceResourceName
+				)
+		);
+
+		let reportErrorToCheck = reportError;
+		if ( isSiteKitAudience && isSiteKitAudiencePartialData ) {
+			reportErrorToCheck = siteKitAudiencesReportError;
+		}
+
+		if ( reportErrorToCheck || totalPageviewsReportError ) {
```

Comment:
It's possible that _both_ `report` and `siteKitAudiencesReport` could be relevant for the current set of audiences, if we have both Site Kit and non-Site Kit audiences selected.

We need to update `useAudienceTilesReports()` to ensure that only the relevant reports are being run, as currently it's always requesting both reports. We could then use the presence of each report to determine if its error needs to be tested - something like this (untested):

Here, though, we could probably just check if either `reportError` or `siteKitAudiencesReportError` is truthy:

```suggestion
		const mainReportErrors = [];

		if ( report ) {
			mainReportErrors.push( reportError );
		}

		if ( siteKitAudiencesReport ) {
			mainReportErrors.push( siteKitAudiencesReportError );
		}

		if ( mainReportErrors.every( Boolean ) || totalPageviewsReportError ) {
```